db/migrate/20150413194512_remove_rendered_notes.rb

```diff
@@ -0,0 +1,29 @@
+class RemoveRenderedNotes < ActiveRecord::Migration
+  def self.up
+    remove_column :todos, 'rendered_notes'
+  end
+
+  def self.down
+    add_column :todos, 'rendered_notes', :text
+    
+    Todo.reset_column_information
+
+    say "Clearing show_from dates from completed todos"
```

Comment:
The thing that confuses me about this migration is that we're clearing the `show_from` dates for completed todos, which doesn't seem strictly necessary. Is this something that just needs to be done for data cleanliness and doesn't really need to be in this PR?